pkg/readiness/ready_tracker.go

```diff
@@ -90,7 +91,8 @@ type Tracker struct {
 
 // NewTracker creates a new Tracker and initializes the internal trackers.
 func NewTracker(lister Lister, mutationEnabled, externalDataEnabled, expansionEnabled bool) *Tracker {
-	return newTracker(lister, mutationEnabled, externalDataEnabled, expansionEnabled, *crashOnFailureFetchingExpectations, nil, nil)
+	// Dereference crashOnFailureFetchingExpectations when we change crashOnFailureFetchingExpectations back to a flag
```

Comment:
As a result of how I set crashOnFailureFetchingExpectations above, I added a comment to change this when we switch back to a flag, but maybe I need to add a TODO? not sure if just a comment is sufficient. @maxsmythe

Reply:
Add a TODO as a comment and open issue to track so we remember to add the flag back would be great!

Reply:
Done. Is a plain TODO sufficient or is there a way to reference/embed an issue to it. Here is the issue I created: #3460